changelog.md

```diff
@@ -0,0 +1,19 @@
+## Changes for 2023-09-27
+
+### Breaking Changes
+
+- Breaking: `/purge-tbox-cache` endpoint removed, `reset-tbox-cache` endpoint added. This reflects that prez now
+  includes a number of common ontologies by default (prez/reference_data/context_ontologies), and on startup will load
+  annotation triples (e.g. x rdfs:label y) from these. As such, the tbox or annotation cache is no longer completely
+  purged but can be reset to this default state instead.
```

Comment:
Probably remove this since you've reverted the api endpoint name back to the old name.

Reply:
done